src/ChoiceList.re

```diff
@@ -32,41 +38,87 @@ let make = (_children) => {
       ReasonReact.Update({...state, label: Some(choice)})
       } : ReasonReact.Update({...state, label: None})
     | UpdateInput(s) => ReasonReact.Update({...state, input: s})
+    | Weissbier => ReasonReact.Update({...state, labels: ["Weissbier", ...state.labels]})
+    | UltimateWeissbier => ReasonReact.Update({...state, weissbier: changeThisBoolPlease(state.weissbier)})
     },
 
   render: self => {
     let new_label = self.state.input;
     let message =
-      "Add: " ++ new_label;
-    <div>
+      "Add";
+    let destinatorstring = "Destinator";
```

Comment:
destinator_title

Reply:
wrote util.Str("Destinator") -> deleted this line